test/parse-result.js

```diff
@@ -220,6 +220,10 @@ describe('Parse result namespace', () => {
       expect(sourceMaps.first().toValue()).to.deep.equal([[1, 2]]);
     });
 
+    it('should have a convenience method for retrieving source map', () => {
+      expect(element.sourceMap.first().toValue()).to.deep.equal([[1, 2]]);
```

Comment:
That is my major concern. But I think we can break this method when do that big change.